CPT331.Data.Parsers/NtKmlParser.cs

```diff
@@ -50,18 +50,18 @@ protected override void OnParse(string fileName, List<Coordinate> coordinates)
 
 				foreach (XmlNode coordinateXmlNode in coordinateXmlNodes)
 				{
-					coordinateValues = $"{coordinateValues} {coordinateXmlNode.InnerText}";
+					coordinateValues = $"{coordinateValues} {coordinateXmlNode.InnerText}".Trim();
```

Comment:
Quick fix for a new NT parser run time error. 

As it's not code that runs a lot in production, I think it's okay.